### ISSUE TITLE
Opportunity to sort vocabulary item requests by number of existing results

### ISSUE THREAD

**Issue by Guybrush88** [Tatoeba/tatoeba2#3130]
I'm browsing the vocabulary item requests for Italian (https://tatoeba.org/it/vocabulary/add_sentences/ita), and currently the list is automatically sorted from items with the least existing results to the ones with the most existing results.

**Idea**
My idea is that I could also reverse the order of the items (for example from the items with the most existing results to the ones with the least existing results), so that I can decide what to contribute first, and this can be helpful if the list of items gets pretty long.

**Comment by ckjpn** [comment]
I agree that it would be a good idea improve these pages, a reverse sort would likely be easy to offer and would help improve the situation.

At this point, there are so many extremely low-frequency words at the top of the English that it is not much use for someone who wants to contribute natural-sounding sentences that would benefit a large number of students.

https://tatoeba.org/en/vocabulary/add_sentences/eng

Here are 2 ideas which might help increase the usefulness of these pages and get more people to use them.

1.
One idea would be adding the possibility to sort by date added, so one could show the most recent requests which would likely be by currently contributing members.  This would be more motivating, since there is a higher likelyhood of your new sentence getting translated.

2. 
Perhaps we could have a way to not display vocabulary requests that have fewer than a certain number of sentences, so members could help increase the number of examples using vocabulary that isn't represented many times in our corpus, but do have a number of sentences.  In other words, let a member choose a number, for example, 2, and then display vocabulary items that only have 2 or more sentences in our database with that vocabulary.
